### ISSUE TITLE
4 CRITICAL, 29 HIGH, 13 MEDIUM, 3 LOW, 5 UNKNOWN CVEs on latest image

### ISSUE THREAD

**Issue by nsvijay04b1** [abutaha/aws-es-proxy#125]
Hi,

*  Please release official docker tag 1.3 ( missing in dockerhub)
*  Please fix below CVEs as customers are concerned to use the image.

Total CVEs as on 23 Nov 2022.

4 CRITICAL, 29 HIGH, 13 MEDIUM, 3 LOW, 5 UNKNOWN CVEs on latest image

Thank you.

```
$ trivy image abutaha/aws-es-proxy:v1.1

abutaha/aws-es-proxy:v1.1 (alpine 3.11.5)

Total: 45 (UNKNOWN: 0, LOW: 2, MEDIUM: 10, HIGH: 29, CRITICAL: 4)

usr/local/bin/aws-es-proxy (gobinary)

Total: 9 (UNKNOWN: 5, LOW: 1, MEDIUM: 3, HIGH: 0, CRITICAL: 0)
```

**Comment by nsvijay04b1** [comment]
#126 PR raised to fix the CVEs.  Please review and accept the change.

**Comment by abutaha** [comment]
Fixed this in the latest release (v1.5)
